docs/conf.py

```diff
@@ -54,17 +54,17 @@
 
 # General information about the project.
 project = u'CC3D Reference Manual'
-copyright = u'2020, Maciej H. Swat, Julio Belmonte, T.J. Sego,  James A. Glazier'
+copyright = u'2024, Maciej H. Swat, Julio Belmonte, T.J. Sego, Pete Fyffe, James A. Glazier'
 author = u'Maciej H. Swat, Julio Belmonte, T.J. Sego, Pete Fyffe, James A. Glazier'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'4.5.0'
+version = u'4.6.0'
 # The full version, including alpha/beta/rc tags.
-release = u'4.5.0'
+release = u'4.6.0'
```

Comment:
This was one of the few files that needed to be adjusted. All of the other work for synchronizing the manuals had been done this winter.